scss/modules/_bp3-overwrites.scss

```diff
@@ -88,4 +95,21 @@
 .bp3-menu {
   background-color: var(--bp3MenuBg);
   color: var(--bp3MenuText);
+}
+
+.bp3-popover .bp3-popover-content {
+  background: unset;
+}
+
+.bp3-popover {
+  -webkit-box-shadow: none;
+  box-shadow: 0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2); // TODO include color into theming
```

Comment:
We maybe need to simplify this and make it themeable at some point. But not really sure how (besides having multiple variables for this)

Reply:
maybe we can set some of these problematic styles in an scss override in theme data like defining a value in the raw section:
(semi pseudo-code)
`defaultTheme.shadowColor = "rgb(16, 22, 26)"`
```
bp3PopoverBoxShadow: undefinedGuard (
theme.shadowColor, c => `0 0 0 1px ${Color(c).alpha(0.1).rgb}, 0 2px 4px ${Color(c).alpha(0.2).rgb}, 0 8px 24px ${Color(c).alpha(0.2).rgb}`
),
```
and then while setting the css for the variables:
```
.bp3-popover {
  -webkit-box-shadow: none;
  box-shadow: ${themeData.bp3PopoverBoxShadow};
}
```